src/cmd/gz.hh

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (C) 2025 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+#ifndef SDF_GZ_HH_
+#define SDF_GZ_HH_
+
+#include <cstring>
+
+#include <sdf/sdf_config.h>
+#include "sdf/system_util.hh"
+
+/// \brief External hook to execute 'gz sdf -k' from the command line.
+/// \param[in] _path Path to the SDF file to validate.
+/// \return Zero on success, negative one otherwise.
+int cmdCheck(const char *_path);
```

Comment:
Can you wrap those in the `sdf` namespace? You can follow the pattern from many of the .hh files in this library.